packages/lit-dev-content/site/tutorials/content/working-with-lists/01.md

```diff
@@ -0,0 +1,55 @@
+Sometimes there may be a need to render a list from something other than an
```

Comment:
I'd mention "iterable" in here, since that's what Lit supports, so like "render a list from another type of iterable, such as a `Set`"